<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRKGastroTableManager</class>
 <widget class="QWidget" name="QRKGastroTableManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Räume und Tische Manager</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <item>
        <widget class="QGroupBox" name="roomsBox">
         <property name="title">
          <string>Räume</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QrkPushButton" name="changeRoomButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Raum bearbeiten</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/edit.png</normaloff>:/src/icons/edit.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QrkPushButton" name="deleteRoomButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Raum löschen</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/eraser.png</normaloff>:/src/icons/eraser.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QrkPushButton" name="newRoomButton">
                <property name="text">
                 <string>Neuer Raum</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/new.png</normaloff>:/src/icons/new.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="roomsScrollArea">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>256</width>
                 <height>440</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListView" name="roomsView"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tablesBox">
         <property name="title">
          <string>Tische</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QScrollArea" name="tablesScrollArea">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>272</width>
                 <height>440</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListView" name="tablesView"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QrkPushButton" name="changeTableButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Tisch bearbeiten</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/edit.png</normaloff>:/src/icons/edit.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QrkPushButton" name="deleteTableButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Tisch löschen</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/eraser.png</normaloff>:/src/icons/eraser.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QrkPushButton" name="newTableButton">
                <property name="text">
                 <string>Neuer Tisch</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../qrk.qrc">
                  <normaloff>:/src/icons/new.png</normaloff>:/src/icons/new.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QrkPushButton" name="exitButton">
         <property name="text">
          <string>Beenden</string>
         </property>
         <property name="icon">
          <iconset resource="../../../qrk.qrc">
           <normaloff>:/src/icons/ok.png</normaloff>:/src/icons/ok.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newRoomButton</tabstop>
  <tabstop>deleteRoomButton</tabstop>
  <tabstop>changeRoomButton</tabstop>
  <tabstop>newTableButton</tabstop>
  <tabstop>deleteTableButton</tabstop>
  <tabstop>changeTableButton</tabstop>
  <tabstop>roomsScrollArea</tabstop>
  <tabstop>tablesScrollArea</tabstop>
  <tabstop>tablesView</tabstop>
 </tabstops>
 <resources>
  <include location="../../../qrk.qrc"/>
  <include location="../../../qrk.qrc"/>
 </resources>
 <connections/>
</ui>
